frappe/public/scss/desk/scrollbar.scss

```diff
@@ -23,7 +18,12 @@ html {
 	background: var(--scrollbar-track-color);
 }
 
-body::-webkit-scrollbar {
-	width: unset;
-	height: unset;
+*::-webkit-scrollbar {
+	width: 6px;
+	height: 6px;
+}
+
+body::-webkit-scrollbar, .modal::-webkit-scrollbar {
+	width: 12px;
+	height: 12px;
```

Comment:
This was changed to `12px` to be consistent with Firefox. (Defaults to `15px` on Google Chrome)